spec/abci++/abci++_tmint_expected_behavior.md

```diff
@@ -56,22 +59,23 @@ consensus-exec      = (inf)consensus-height
 consensus-height    = *consensus-round decide commit
 consensus-round     = proposer / non-proposer
 
-proposer            = *got-vote prepare-proposal *got-vote process-proposal [extend]
-extend              = *got-vote extend-vote *got-vote
-non-proposer        = *got-vote [process-proposal] [extend]
+proposer            = *got-vote prepare-proposal *got-vote process-proposal
+non-proposer        = *got-vote [process-proposal]
```

Comment:
We won't see "got-vote", as we don't have vote extensions, you can delete all occurrences